patches/@rrweb__record@2.0.0-alpha.18.patch

```diff
@@ -1,8 +1,55 @@
 diff --git a/dist/record.js b/dist/record.js
-index d9c57625633fb87da27fc5948c1b15a7c4e4caa5..1932f81f2e330cc904e8349a74911e6b3c12669a 100644
+index d9c57625633fb87da27fc5948c1b15a7c4e4caa5..5d583e81bb7b7e49b7a091ca0c217a37116861b1 100644
 --- a/dist/record.js
 +++ b/dist/record.js
-@@ -68,10 +68,10 @@ function getUntaintedPrototype$1(key) {
+@@ -26,14 +26,33 @@ const testableMethods$1 = {
+   Element: [],
+   MutationObserver: ["constructor"]
+ };
++/*
++Angular zone patches many things and can pass the untainted checks below, causing performance issues
++Angular zone, puts the unpatched originals on the window, and the names for hose on the zone object.
++So, we get the unpatched versions from the window object if they exist.
++You can rename Zone, but this is a good enough proxy to avoid going to an iframe to get the untainted versions.
++see: https://github.com/angular/angular/issues/26948
++*/
++function angularZoneUnpatchedAlternative$1(key) {
++  const angularUnpatchedVersionSymbol = (
++    globalThis
```

Comment:
angular puts a function on Zone you can call with the name of the thing
and you get back undefined or where it has stored the thing